src/encounters/LuanaEncounter.js

```diff
@@ -26,7 +64,28 @@ class LuanaEncounter extends Encounter {
   }
 
   getInteractions() {
+    const { activeSprite } = this.props;
+    const { sprite } = this.state;
+    const REWARD = 20;
+    let nextGoal = goalSprite();
+    while (nextGoal.species === sprite.species && nextGoal.color === sprite.color) {
```

Comment:
Although there's only one interaction, it'd be better to generate the nextGoal when the player runs the interaction rather than at the time of computing the interactions.  Note that getInteractions() runs on every re-render for example.